public/index.html

```diff
@@ -7,7 +7,7 @@
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
-      content="Simple probability calculator for cubes (Maplestory), now with Daybreak Pendant Simulator"
+      content="Web site created using create-react-app"
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
```

Comment:
Please revert the logo to the horribly drawn cube instead of using the default react logo